Pre-empt user from running -local if root (#2731)
The-OpenROAD-Project/OpenROAD: README.md

``` shell
./etc/DependencyInstaller.sh -help

> Usage: ./etc/DependencyInstaller.sh                # Installs all of OpenROAD's dependencies no need to run -base or -common.
>                                                        # Requires priviledged access.
>        ./etc/DependencyInstaller.sh -base          # Installs OpenROAD's dependencies using package managers 
>                                                        # (-common must be executed in another command)
>        ./etc/DependencyInstaller.sh -common        # Installs OpenROAD's common dependencies (-base must be executed in another command)
>        ./etc/DependencyInstaller.sh -prefix=DIR    # Installs dependencies in an existing user specified directory
>        ./etc/DependencyInstaller.sh -local         # Installs common dependencies in "/home/habibayassin/.local". Only used with 
>                                                        # -common. Root access is still needed for other dependencies. Flag cannot be used # with sudo or with root access

Reviewer: ```suggestion
Usage: etc/DependencyInstaller.sh
                                # Installs all of OpenROAD's dependencies no
                                #     need to run -base or -common. Requires
                                #     priviledged access.
                                #
       etc/DependencyInstaller.sh -base
                                # Installs OpenROAD's dependencies using
                                #     package managers (-common must be
                                #     executed in another command).

       etc/DependencyInstaller.sh -common
                                # Installs OpenROAD's common dependencies
                                #     (-base must be executed in another
                                #     command).

       etc/DependencyInstaller.sh -prefix=DIR
                                # Installs common dependencies in an existing
                                #     user-specified directory. Only used
                                #     with -common. This flag cannot be used
                                #     with sudo or with root access.

       etc/DependencyInstaller.sh -local
                                # Installs common dependencies in
                                #    "$HOME/.local". Only used with
                                #    -common. This flag cannot be used with
                                #    sudo or with root access.
```
